gabbi/handlers/jsonhandler.py

```diff
@@ -13,13 +13,17 @@
 """JSON-related content handling."""
 
 import json
+import re
 
 import six
 
 from gabbi.handlers import base
 from gabbi import json_parser
 
 
+CONTENT_TYPE = re.compile('^application/json[^a-zA-Z0-9]*$')
```

Comment:
I guess your reasoning here is "this matches `json` and anything like `json+*`, `json-*`, etc"?

The presupposes that anything `jsonfoo` is not actually json, or is a bad name, which is probably a pretty good assumption. I couldn't find any registered mimetypes that would be an issue.

Reply:
Yes, the assumption was that "json" followed by alphanumeric characters ain't JSON-as-currently-defined.

No, I didn't think about this in such detail that I explicitly considered `json-[+-]` and such; I just wanted to denylist the above. Seems sensible now though?